components/blitz/src/omero/client.java

```diff
@@ -329,7 +329,10 @@ private void init(Ice.InitializationData id) {
 
         // Strictly necessary for this class to work
         optionallySetProperty(id, "Ice.ImplicitContext", "Shared");
-        if (Ice.Util.intVersion() <= 30500) {
+        if (Ice.Util.intVersion() >= 30600) {
+            optionallySetProperty(id, "Ice.ACM.Client.Timeout", "60000");
```

Comment:
For the pages in general, we could look into that. 
even adding note for dev will be useful e.g. ```Trace.Network```, ```Trace.Security```